### ISSUE TITLE
Consistent Modeling of Direct Communication Paths

### ISSUE THREAD

**Issue by trieloff** [adobe/xdm#94]
For every channel that allows outbound communication (email, phone, sms, …) we should have a consistent way of expressing ways to reach a customer.

I'd suggest we introduce a `xdm:communication` object in place of existing `xdm:email`, `xdm:homePhone`, `xdm:workPhone`, etc. that is keyed with the channel name. 

The email address, address, phonenumber schemas would get a common superschema with fields like primary, opt-in status, delivery status, etc.

**Comment by cdegroot-adobe** [comment]
Agreed, lets fix this for 1.0. I will take up the task of a PR for it.
